lib/business_calendar.rb

```diff
@@ -32,5 +32,6 @@ def load_config
 end
 
 require 'yaml'
+require 'business_calendar/shim'
```

Comment:
Only require if necessary? I know there's a guard in the file, but it's prolly clearer to just avoid requiring it at all.

In fact, maybe `shim` should be something egregious like `shim_for_ree_ruby18_compat`. =)